site/docs/install-windows.md

```diff
@@ -3,64 +3,164 @@ layout: documentation
 title: Installing Bazel on Windows
 ---
 
-# <a name="windows"></a>Installing Bazel on Windows
+# Installing Bazel on Windows
 
-### Prerequisites
+## 1. Check your system
 
-*   64 bit Windows 7 or newer, or 64 bit Windows Server 2008 R2 or newer
+Recommended: 64 bit Windows 10, version 1703 (Creators Update) or newer, enable "Developer Mode".
 
-*   [Microsoft Visual C++ Redistributable for Visual Studio 2015](https://www.microsoft.com/en-us/download/details.aspx?id=48145)
+<!-- Developer mode: for symlink support. -->
 
-    These are common DLLs that Bazel needs. You may already have them installed.
+Also supported:
 
-Bazel 0.24.x and older:
+*   64 bit Windows 7 or newer
 
-*   [MSYS2 shell](https://msys2.github.io/)
+*   64 bit Windows Server 2008 R2 or newer
 
-Bazel 0.25.0 and newer:
+## 2. Install the prerequisites
 
-*   MSYS2 is not always required. Read about [Using Bazel without Bash](windows.html#using-bazel-without-bash).
+*   [Visual C++ Redistributable for Visual Studio 2015](https://www.microsoft.com/en-us/download/details.aspx?id=48145)
 
-### Getting Bazel
+*   [MSYS2 x86_64](https://www.msys2.org/)
 
-1.  [Download Bazel for Windows from GitHub](https://github.com/bazelbuild/bazel/releases).
+    You should use the default install path.
 
-    Look for `bazel-<version>-windows-x86_64.exe`, for example
-    `bazel-0.16.1-windows-x86_64.exe`.
+*   MSYS2 packages
 
-    **Tip:** Rename the binary to `bazel.exe` and move it to a directory on your
-    `%PATH%` (for example to `c:\bazel`), so you can run Bazel by typing `bazel`
-    in any directory.
+    Open the MSYS2 terminal, and run this command:
 
-1.  Edit environment variables.
+    ```
+    pacman -S zip unzip patch diffutils git
+    ```
 
-    Open the "Environment Variables" dialog box from Control Panel or Start
-    Menu, and add or edit the following variables under the "User variables"
-    section:
-    1.  **Edit `Path`**. Add new entries to the beginning of the list:
-        *   The directory of `bazel.exe`. (Example: `c:\bazel`).
-        *   The `usr\bin` directory of MSYS2. (Example: `c:\msys64\usr\bin`).
-        *   If you will build **Python** code: the directory of `python.exe`.
-            (Example: `c:\python3`).
-    1.  **Add `BAZEL_SH`**. Its value must be the path to MSYS2 Bash.
-        Example: `c:\msys64\usr\bin\bash.exe`
-    1.  **Add `JAVA_HOME`** (if you will build **Java** code). Its value must be
-        the directory where you installed the Java JDK 8, for example
-        `C:\Program Files\Java\jdk1.8.0_152`. In order to use this with the
-        default local_jdk javabase, it must be installed on a volume which
-        Windows considers to be **local**, network mounted filesystems will not
-        work.
+*   [Build Tools for Visual Studio 2019](https://aka.ms/buildtools)
```

Comment:
Why VS 2019 is a prerequisite of Bazel? I think it should be moved to `Build C++ with Bazel` section

Reply:
You're correct. However, I think many users will need it.

I focused on simple instructions, at the cost of being correct. Now it's one decision less that the user has to make, which I think is a worthy tradeoff.

Otherwise they'd have to consider whether they'll need VC++... and the install instructions would be spread across more docs.

Reply:
I want the instructions to be as simple and as short as possible. It should be a well-lit path for users who just want a basic, well working Bazel setup.

Reply:
I see, but I think it's better to mention Bazel also works without VC installed if users are not building C++ targets.

Reply:
Done. I created a new "optional" section.

Reply:
Nice, thank you!!